tutorials/SpectroscopicDataReductionBasics/1-SpectroscopicTraceTutorial.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
<p>Let's try fitting with degree 3 instead of just 2.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>rephrased</p>